packages/web/src/components/track/DownloadRow.module.css

```diff
@@ -0,0 +1,9 @@
+.spinner {
+  width: var(--unit-4) !important;
```

Comment:
didn't work without !important and tbh i didn't spend too much time digging for why